src/ubuntu/20.04/webassembly/Dockerfile

```diff
@@ -49,7 +49,7 @@ RUN curl -sSL https://netcorenativeassets.blob.core.windows.net/resource-package
     && chmod +x /usr/local/bin/v8
 
 # Install Wasi toolchain
-ENV WASI_SDK_VERSION=20
+ENV WASI_SDK_VERSION=21
```

Comment:
I don't think we use this container anymore actually. We could probably delete it.